script/build/windows.ps1

```diff
@@ -6,7 +6,7 @@
 #
 #        http://git-scm.com/download/win
 #
-# 2. Install Python 3.7.x:
+# 2. Install Python 3.9.x:
 #
 #        https://www.python.org/downloads/
 #
```

Comment:
There are more py37 refs below in this file.

Reply:
We'll need to make sure that we bump the CI machine's Python version as well